docs/features/configExport.md

```diff
@@ -0,0 +1,55 @@
+# Exporting configurations
+
+The plugin supports exporting existing configurations as YAML.
+This is a feature available to the Jenkins administrators under the `http://[your_jenkins_url]/configuration-as-code/` URL.
+
+Export feature is **NOT** intended to offer a directly usable jenkins.yaml configuration. 
+It can be used for inspiration writing your own production-ready YAML, but be aware that export can be partial, 
```

Comment:
as far as I know this doesn't happen that often now, all the common failures have been fixed